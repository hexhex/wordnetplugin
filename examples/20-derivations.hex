%% this returns all words, of which a given word
%% is an attribute
%% POS: adjectives

%% the search number for derivations
%% is 20.

word("nice").
derivations(W,SW,D) :- &wordnet[20,W](_,SW,D,_), word(W).

derivations("nice",1,"niceness"), 
derivations("nice",2,"politeness"), 
derivations("nice",2,"niceness"), 
derivations("nice",5,"politeness"), 
derivations("nice",5,"niceness")?
