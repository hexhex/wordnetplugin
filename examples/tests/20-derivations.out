{derivations("nice",1,"niceness"), derivations("nice",2,"politeness"), derivations("nice",2,"niceness"), derivations("nice",5,"politeness"), derivations("nice",5,"niceness"), word("nice")}
